Assets/Scenes/Gyms/GYM_Highlight/GYM_Highlight.unity

```diff
@@ -0,0 +1,3702 @@
+%YAML 1.1
```

Comment:
Move these GYMs to the correct subfolder that has been created in the latest develop branch